cmd/apps/kuma_app.go

```diff
@@ -0,0 +1,117 @@
+package apps
+
+import (
+	"fmt"
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallKuma() *cobra.Command {
+	var kuma = &cobra.Command{
+		Use:          "kuma",
+		Short:        "Install Kuma",
+		Long:         "Install Kuma",
+		Example:      ` arkade app install kuma`,
+		SilenceUsage: true,
+	}
+
+	kuma.Flags().String("namespace", "kuma-system", "Namespace for the app")
+
+	kuma.Flags().String("control-plane-mode", "standalone", "Kuma CP modes: one of standalone,zone,global")
+	kuma.Flags().Bool("auto-scale", false, "Enable Horizontal Pod Autoscaling (requires the Metrics Server)")
+
+	kuma.Flags().Bool("use-cni", false, "Use CNI instead of proxy init container")
+	kuma.Flags().Bool("ingress", false, "Deploy Ingress for cross cluster communication")
+
+	kuma.Flags().StringArray("set", []string{}, "Use custom flags or override existing flags \n(example --set ingress.enabled=false)")
+
+	kuma.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		if err := config.SetKubeconfig(kubeConfigPath); err != nil {
+			return err
+		}
+
+		arch := k8s.GetNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+		if suffix := getValuesSuffix(arch); suffix == "-armhf" {
+			return fmt.Errorf(`Kuma is currently not supported on armhf architectures`)
+		}
+
+		namespace, _ := command.Flags().GetString("namespace")
+		controlPlaneMode, _ := command.Flags().GetString("control-plane-mode")
+		autoScale, _ := command.Flags().GetBool("auto-scale")
+		useCNI, _ := command.Flags().GetBool("use-cni")
+		ingress, _ := command.Flags().GetBool("ingress")
+		customFlags, _ := command.Flags().GetStringArray("set")
+
+		overrides := map[string]string{}
+
+		// control-plane-mode  standalone,zone,global
+		if controlPlaneMode == "standalone" || controlPlaneMode == "zone" || controlPlaneMode == "global" {
+			overrides["controlPlane.mode"] = controlPlaneMode
+		} else {
+			return fmt.Errorf(`Kuma's control-plane mode must be one of global, zone or standalone.`)
```

Comment:
All errors need to start with a lowercase in Go

Reply:
Sure, sorry about that, will be fixed in a minute, I'll amend it to avoid the rebase process later.